lib/posix/posix_internal.h

```diff
@@ -95,24 +68,9 @@ static inline uint32_t mark_pthread_obj_uninitialized(uint32_t obj)
 	return obj & ~PTHREAD_OBJ_MASK_INIT;
 }
 
-struct posix_thread *to_posix_thread(pthread_t pthread);
+struct posix_thread *to_posix_thread(pthread_t pth);
 
 /* get and possibly initialize a posix_mutex */
-struct posix_mutex *to_posix_mutex(pthread_mutex_t *mu);
-
-/* get a previously initialized posix_mutex */
-struct posix_mutex *get_posix_mutex(pthread_mutex_t mut);
-
-/* get and possibly initialize a posix_cond */
-struct posix_cond *to_posix_cond(pthread_cond_t *cvar);
-
-/* get a previously initialized posix_cond */
-struct posix_cond *get_posix_cond(pthread_cond_t cond);
-
-/* get and possibly initialize a posix_key */
-pthread_key_obj *to_posix_key(pthread_key_t *keyp);
-
-/* get a previously initialized posix_key */
-pthread_key_obj *get_posix_key(pthread_key_t key);
+struct k_mutex *to_posix_mutex(pthread_mutex_t *mu);
```

Comment:
Shouldn't we change the name here? Previously this function did conversion to posix mutex, but now it converts to regular Zephyr mutex.

Bumped into this reviewing later code - as due to `to_posix_mutex` function name code looks incorrect.

Reply:
I didn't feel the need to rename functions / introduce additional noise.

Reply:
@evgeniy-paltsev - I would rather not change the pattern with these here. There are similar functions throughout the posix code and they are internal.